padcli/command/initcmd.go

```diff
@@ -146,28 +152,28 @@ func initConfig(ctx context.Context, path string) error {
 			"https://www.jetpack.io/launchpad/docs/reference/launchpad.yaml-reference/ \n",
 		configPath,
 	)
+
 	return nil
 }
 
 func runConfigSurvey(
 	ctx context.Context,
+	authProvider provider.Auth,
 	appName string,
 ) (*SurveyAnswers, error) {
 
 	clusters, err := cmdOpts.ClusterProvider().GetAll(ctx)
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
-	if len(clusters) == 0 {
-		// No clusters available. Show user-friendly error.
-		return nil, errorutil.NewUserError("We were unable to find a kubernetes cluster in your kubeconfig, " +
-			"which is required to use Launchpad. You can set one up with Docker or use Launchpad to create one for you")
-	}
 
 	clusterNames := []string{}
 	for _, c := range clusters {
 		clusterNames = append(clusterNames, c.GetName())
 	}
+	// In case user wants to log in and use a jetpack managed cluster.
+	clusterNames = append(clusterNames, jetpackManagedCluster)
```

Comment:
but, now we may have both `jetpack-cloud` and `Jetpack managed cluster` as options, which doesn't make sense.

Reply:
why don't we remove the `Jetpack managed cluster` option and then just indicate which clusters are jetpack-managed from the list of clusters we already have?

Reply:
Because if a commercial user is not logged in, we won't have the full set of clusters. By selecting `Jetpack managed cluster` we will drop them in the login flow, knowing that they are not OSS users.

Reply:
> we won't have the full set of clusters

but we will, because they were (likely) previously merged into their kubeconfig

Reply:
If we do have a full set, then they can simply select it from the previous question. I'll give you a full demo